hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataStorage.java

```diff
@@ -388,6 +388,11 @@ static int getParallelVolumeLoadThreadsNum(int dataDirs, Configuration conf) {
     try {
       final List<StorageLocation> successLocations = loadDataStorage(
           datanode, nsInfo, dataDirs, startOpt, executor);
+
+      if (successLocations.isEmpty()) {
```

Comment:
Can we adjust the JIRA title to say something like this?
We should also have a test.

Reply:
thank you very much.
I have modified the source code and resubmitted it.
The title has been modified.